src/subscription_manager/utils.py

```diff
@@ -161,7 +161,7 @@ def is_valid_server_info(conn):
         return False
 
 
-def is_owner_using_golden_ticket(uep=None, identity=None, owner=None):
+def is_simple_content_access(uep=None, identity=None, owner=None):
     """
     This function return True, when current owner uses contentAccessMode equal to org_environment. This
```

Comment:
Is the value of the contentAccessMode still "org_environment"?

Reply:
Yes